scripts/hooks/pre-commit.sh

```diff
@@ -71,7 +71,7 @@ run_kotlin_linter() {
   java -jar "$detekt_cli_jar" \
     --plugins "$detekt_plugins_input" \
     --config "$detekt_dir/config.yml" \
-    --jvm-target "1.8" \
+    --jvm-target "11" \
```

Comment:
Could you explain why this is needed/preferred? Shouldn't we keep it aligned with the rest of the project?

Reply:
> Shouldn't we keep it aligned with the rest of the project?
Yes, the rest of the project is using 11 as the JVM target

Reply:
Oh, I see; my bad 🫠

Reply:
I don't think we'll ever increase our JVM target but maybe in future it could be helpful to read this from our JvmOptions.